components/accept-financial-contributions/StripeOrBankAccountPicker.js

```diff
@@ -3,24 +3,24 @@ import PropTypes from 'prop-types';
 import { Box, Flex } from '../Grid';
 import { FormattedMessage, defineMessages, injectIntl } from 'react-intl';
 import styled from 'styled-components';
-import { graphql } from '@apollo/react-hoc';
 import { has, find } from 'lodash';
 import themeGet from '@styled-system/theme-get';
 import { CheckboxChecked } from '@styled-icons/boxicons-regular/CheckboxChecked';
+import { graphql } from '@apollo/react-hoc';
+import { withRouter } from 'next/router';
+
+import { connectAccount } from '../../lib/api';
+import { Router } from '../../server/pages';
+import { API_V2_CONTEXT, gqlV2 } from '../../lib/graphql/helpers';
```

Comment:
Adding GraphQL API v2 and re-ordering imports (that's gonna be done for the whole codebase soon).